x/alliance/keeper/grpc_query.go

```diff
@@ -16,6 +17,127 @@ type QueryServer struct {
 	Keeper
 }
 
+func (k QueryServer) AllAlliancesDelegations(c context.Context, req *types.QueryAllAlliancesDelegationsRequest) (*types.QueryAlliancesDelegationsResponse, error) {
```

Comment:
What do you think about giving a name to the return types to avoid the necessity of creating the types inline with **nil** values?

> the comment applies to line 20, 65 and 97

Reply:
I think thats a good idea but we just need to be consistent throughout the codebase.